conans/client/conan_api.py

```diff
@@ -406,12 +406,14 @@ def download(self, reference, remote_name=None, package=None, recipe=False):
     def install_reference(self, reference, settings=None, options=None, env=None,
                           remote_name=None, verify=None, manifests=None,
                           manifests_interactive=None, build=None, profile_name=None,
-                          update=False, generators=None, install_folder=None, cwd=None):
+                          update=False, generators=None, install_folder=None,
+                          deploy_folder=None, cwd=None):
 
         try:
             recorder = ActionRecorder()
             cwd = cwd or os.getcwd()
             install_folder = _make_abs_path(install_folder, cwd)
+            deploy_folder = _make_abs_path(deploy_folder, cwd)
```

Comment:
We should keep it to None when None

Reply:
no, install folder always exists (defaulted to cwd or whatever) but for `deploy_folder` we need the `None` possibility to know if we want to run or not the default deployment.